tests/integration/tier0/test-check-rollback-handling/test_check_rollback_handling.py

```diff
@@ -104,3 +101,111 @@ def test_check_untrack_pkgs_force(convert2rhel, shell):
         assert c2r.exitstatus != 0
 
     post_rollback_check(shell)
+
+
+def terminate_and_assert_good_rollback(c2r):
```

Comment:
Move this function more above among the others functions defintions.

Reply:
Done.

Reply:
Looks better, will do.
Thanks.